Renewed/nl_nl.json

```
@@ -322,7 +322,7 @@
 	"block.lotr.orc_torch": "Orkfakkel",
 	"block.lotr.orc_bars": "Orkstalen tralies",
 	"block.lotr.dwarven_bars": "Dwergenstalen tralies",
-	"block.lotr.bronze_bars": "Bronsn tralies",
+	"block.lotr.bronze_bars": "Bronsen tralies",
```

Naar mijn mening is het toch echt `bronzen`
```suggestion
	"block.lotr.bronze_bars": "Bronzen tralies",
```

---

Oeps. ik kan niet een goed een fout verbeteren -_-